datafusion/src/physical_plan/expressions/lead_lag.rs

```diff
@@ -98,20 +105,58 @@ impl BuiltInWindowFunctionExpr for WindowShift {
         Ok(Box::new(WindowShiftEvaluator {
             shift_offset: self.shift_offset,
             values,
+            default_value: self.default_value.clone(),
         }))
     }
 }
 
 pub(crate) struct WindowShiftEvaluator {
     shift_offset: i64,
     values: Vec<ArrayRef>,
+    default_value: Option<ScalarValue>,
+}
+
+fn shift_with_default_value(
```

Comment:
you can add a comment of todo to push this upstream to arrow-rs

Reply:
Added a TODO comment